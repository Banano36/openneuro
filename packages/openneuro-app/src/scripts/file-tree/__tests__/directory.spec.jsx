import React from 'react'
import { shallow } from 'enzyme'
import Directory from '../directory.jsx'

describe('Directory component', () => {
  it('renders with default props', () => {
    expect(shallow(<Directory />)).toMatchSnapshot()
  })
  it('expands when clicked', () => {
    const wrapper = shallow(<Directory name="Top Level" />)
    wrapper.find('button').simulate('click')
    expect(wrapper.find('Panel').hasClass('open')).toBe(true)
  })
})
