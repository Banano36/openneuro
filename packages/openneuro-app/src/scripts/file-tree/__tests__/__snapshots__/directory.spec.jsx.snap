// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directory component renders with default props 1`] = `<Component />`;
